Description: lintian reports several typos in manpages. This patch fixes those
 reported
Author: Paul Gevers <elbrus@debian.org>
Forwarded: http://bugs.motifzone.net/show_bug.cgi?id=1587
Last-reviewed: 14 Jan 2013

--- a/doc/man/man1/mwm.1
+++ b/doc/man/man1/mwm.1
@@ -359,7 +359,7 @@
 (and other associated messages) by saving the geometries of its clients to a state file\&. 
 \fBmwm\fP can then be restarted by the XSMP session manager\&.
 The default location for the state file is \fB$HOME/\&.mwmclientdb\fP\&.
-This location can be overriden with the resource \fBsessionClientDB\fP\&.
+This location can be overridden with the resource \fBsessionClientDB\fP\&.
 .SS "X Resources"
 .PP
 The \fBmwm\fP command is configured from its resource
--- a/doc/man/man3/VendorShell.3
+++ b/doc/man/man3/VendorShell.3
@@ -76,7 +76,7 @@
 to the VendorShell resources for controlling toolTips. VendorShell displays
 the XmNtooTipString in an XmLabel that is a child of a transientShell. The name
 of the transientShell is TipShell, and the name of the XmLabel is TipLabel.
-The appearance of the tip can be controlled by specifing resources on these
+The appearance of the tip can be controlled by specifying resources on these
 widgets.
 .PP
 If an application uses the \fBXmNmwmDecorations\fP,
--- a/doc/man/man3/XmComboBox.3
+++ b/doc/man/man3/XmComboBox.3
@@ -581,7 +581,7 @@
 ComboBox \fBXmNcomboBoxType\fP is
 \fBXmDROP_DOWN_LIST\fP,
 \fB<osfActivate>\fP, \fB<osfCancel>\fP, and
-\fBReturn\fP are overriden by ComboBox actions\&.
+\fBReturn\fP are overridden by ComboBox actions\&.
 .IP "\fB:c\fP <Key>\fB<osfDown>\fP:" 10
 CBDropDownList()
 .IP "\fB:c\fP <Key>\fB<osfUp>\fP:" 10
--- a/doc/man/man3/XmGetPixmap.3
+++ b/doc/man/man3/XmGetPixmap.3
@@ -109,7 +109,7 @@
 In addition to X bitmap files (XBM), Motif also supports XPM (X
 Pixmap) file formats and, from version 2.3, JPEG and PNG image formats\&.
 (Note that support of JPEG and PNG image format is an optional feature
-of Motif, in order to check if current version supports PNG ang JPEG
+of Motif, in order to check if current version supports PNG and JPEG
 image formats the PNG_SUPPORT and JPEG_SUPPORT macros should be checked
 correspondingly.)
 The \fBXBMLANGPATH\fP specifies the path for
--- a/doc/man/man3/XmGetPixmapByDepth.3
+++ b/doc/man/man3/XmGetPixmapByDepth.3
@@ -111,7 +111,7 @@
 In addition to X bitmap files (XBM), Motif also supports XPM (X
 Pixmap) file formats, and, from version 2.3, JPEG and PNG image formats\&.
 (Note that support of JPEG and PNG image format is an optional feature
-of Motif, in order to check if current version supports PNG ang JPEG
+of Motif, in order to check if current version supports PNG and JPEG
 image formats the PNG_SUPPORT and JPEG_SUPPORT macros should be checked
 correspondingly.)
 The \fBXBMLANGPATH\fP specifies the path for
--- a/doc/man/man3/XmHierarchyGetChildNodes.3
+++ b/doc/man/man3/XmHierarchyGetChildNodes.3
@@ -69,8 +69,8 @@
 \fBXmHierarchy\fP(3)\&.
 .SH "RETURN"
 .PP
-Returns WidgetList with child nodes of the hierachy, or NULL
-if the hierachy contains no children\&.
+Returns WidgetList with child nodes of the hierarchy, or NULL
+if the hierarchy contains no children\&.
 .SH "RELATED"
 .PP
 \fBXmHierarchy\fP(3)\&.
--- a/doc/man/man3/XmManager.3
+++ b/doc/man/man3/XmManager.3
@@ -502,7 +502,7 @@
 Indicates whether the posting process should continue\&. The
 application may modify this field\&.
 .IP "\fItarget\fP" 10
-Specifies the most specific widget or gadget that the menu sytem found
+Specifies the most specific widget or gadget that the menu system found
 from the event that matches the event\&.
 .SS "Translations"
 .PP
--- a/doc/man/man3/XmPrimitive.3
+++ b/doc/man/man3/XmPrimitive.3
@@ -609,7 +609,7 @@
 Indicates whether the posting process should continue\&. The
 application may modify this field\&.
 .IP "\fItarget\fP" 10
-Specifies the most specific widget or gadget that the menu sytem found
+Specifies the most specific widget or gadget that the menu system found
 from the event that matches the event\&.
 .SS "Translations"
 .PP
--- a/doc/man/man3/XmRendition.3
+++ b/doc/man/man3/XmRendition.3
@@ -71,7 +71,7 @@
 If a resource in a rendition is unspecified, usually by setting it to
 \fBXmAS_IS\fP or \fBXmUNSPECIFIED_PIXEL\fP,
 then the value to be used for that resource is the value of the
-immediately preceeding rendition in \fBXmString\fR\&. If that value
+immediately preceding rendition in \fBXmString\fR\&. If that value
 is unspecified, then the preceding value is used, and so on\&. If
 no renditions specify a value for a resource, then a default value
 will be used\&.
--- a/doc/man/man3/XmStringGetNextComponent.3
+++ b/doc/man/man3/XmStringGetNextComponent.3
@@ -111,7 +111,7 @@
 \fBXmSTRING_COMPONENT_UNKNOWN\fP\&. The data returned by these
 components is returned in the \fIunknown_tag\fP, \fIunknown_length\fP,
 and \fIunknown_value\fP fields\&. This apparent inconsistency is
-designed to accomodate older applications that may not be equipped to
+designed to accommodate older applications that may not be equipped to
 handle the newer component types of Motif version 2\&.0 and beyond\&.
 Consequently, the use of this procedure is not recommended\&. Instead,
 use the \fBXmStringGetNextTriple\fP procedure, which provides all the
